src/SWP/Component/TemplatesSystem/LICENSE

```diff
@@ -0,0 +1,675 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
The license should be AGPL: https://github.com/superdesk/web-publisher/blob/master/LICENSE.md